README.md

```diff
@@ -80,7 +80,13 @@ Alternatively, you can install the add-ins on your production database individua
                 "ja": "エンジンデータプロフィール"
             },
             "icon": "https://cdn.jsdelivr.net/gh/Geotab/sdk-addin-samples@master/addin-engine-data-button/dist/images/icon.svg"
-        }
+        },
+        {
+			"page": "map",
+			"mapScript": {
+				"url": "https://cdn.jsdelivr.net/gh/Geotab/sdk-addin-samples@master/hos-data-on-live-map/index.html"
```

Comment:
Maybe I was testing incorrectly but I found I got CORS error when trying to test JS deliver map add-in from your fork:
```
Access to XMLHttpRequest at 'https://cdn.jsdelivr.net/gh/life777/sdk-addin-samples@hos-map-add-in/hos-data-on-live-map/index.html' from origin 'https://mypreview.geotab.com' has been blocked by CORS policy: Request header field content-type is not allowed by Access-Control-Allow-Headers in preflight response.
```
Config used:
```
{{ "name": "Compliance Data Summary", "supportEmail": "support@geotab.com", "version": "1.0", "items": [ { "page": "map", "mapScript": { "url": "https://cdn.jsdelivr.net/gh/life777/sdk-addin-samples@hos-map-add-in/hos-data-on-live-map/index.html" } } ], "isSigned": false}
```